test/conftest.py

```diff
@@ -9,50 +8,6 @@
 pynvim.setup_logging("test")
 
 
-@pytest.fixture(autouse=True)
-def cleanup_func(vim):
```

Comment:
can we leave a stub here (with `pass`)?

Reply:
Why? :confused:

Reply:
if I want to add setup code later it will be a pain to find out how. Stubs don't hurt anything.

Not a blocker though.

Reply:
Stubs at least get called all the time (not really that bad, but still).
Should be discoverable through the Git history I think.
(and if you're used to pytest you would know what you need anyway)